README.md

```diff
@@ -86,15 +86,31 @@ Note that non-default networks require extra [firewall setup](https://cloud.goog
   external provisioner can be used to provide similar functionality for
   Kubernetes clusters.
 
+## Deploying the Driver
+
+* Set up a service account with appropriate role binds, and download a service account key. This
+  service account will be used by the driver to provision Filestore instances and otherwise access
+  GCP APIs. This can be done by running `./deploy/project_setup.sh` and pointing to a directory to
+  store the SA key. To prevent your key from leaking do not make this directory publicly
+  accessible!
+  
+```$ PROJECT=<your-gcp-project> GCFS_SA_DIR=<your-directory-to-store-credentials-by-default-home-dir> ./deploy/project_setup.sh```
+
+* Choose a stable overlay that matches your cluster version, eg `stable-1-19`. If you are running a
```

Comment:
maybe we should also clearly mention that prow-gke-release-staging* are not to be used, as this is only for testing purpose.

Reply:
good idea, done